link-check/linkCompare.js

```diff
@@ -2,18 +2,25 @@ import fs from 'fs';
 import { fetch } from '@adobe/fetch';
 import { docx2md } from '@adobe/helix-docx2md';
 
+export function comparison(url1, url2) {
+  return (url1.host === url2.host) && (url1.pathname === url2.pathname);
```

Comment:
no change needed, but I wonder if we even need to compare hosts. I feel like you ran into this where `b.a.com` !== `main--bacom-blog--adobecom.hlx.page/live` and lord knows if even a full url was used.

Reply:
This is an issue with some links and I started to address it but I'm moving that work to a separate ticket as we have already handed off the report.